rel/overlay/etc/default.ini

```diff
@@ -23,6 +23,7 @@ file_compression = snappy
 ; and/or more OS page cache hits, but they can also increase overall response
 ; time for writes when there are many attachment write requests in parallel.
 attachment_stream_buffer_size = 4096
+db_definitions = [couch_dbs, mem3_dbs, couch_replicator_dbs, cassim_dbs]
```

Comment:
It is plural because `couch_define_db` behavior defined multiple dbs ([see](https://github.com/iilyak/couchdb-mem3/blob/2635-systems_dbs/src/mem3_dbs.erl#L8)).